interface/modules/custom_modules/oe-module-weno/src/Services/LogProperties.php

```diff
@@ -151,13 +163,66 @@ public function logSync()
             file_put_contents($this->rxsynclog, $rpt);
             $logstring = "prescription log import initiated successfully";
             EventAuditLogger::instance()->newEvent("prescriptions_log", $_SESSION['authUser'], $_SESSION['authProvider'], 1, "$logstring");
+            $wenolog->insertWenoLog("prescription", "Success");
         } else {
             EventAuditLogger::instance()->newEvent("prescriptions_log", $_SESSION['authUser'], $_SESSION['authProvider'], 1, "$statusCode");
+            $wenolog->insertWenoLog("prescription", "Failed");
+            return false;
         }
 
         if (file_exists($this->rxsynclog)) {
             $log = new LogImportBuild();
             $log->buildInsertArray();
+            return true;
+        }
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getProviderEmail()
+    {
+        if($_SESSION['authUser']){
+            $provider_info = sqlQuery("select setting_value AS email from user_settings where setting_user = ? and setting_label = 'global:weno_provider_email'", [$_SESSION["authUserID"]]);
```

Comment:
user settings are stored in globals (they will overwrite the main global if it exists), so should be able to get above by just using $GLOBALS['weno_provider_email']